projects/extension/src/background/ConnectionManager.ts

```diff
@@ -96,7 +83,7 @@ export class ConnectionManager extends (EventEmitter as { new(): StateEmitter })
   }
 
   /**
-   * disconnectTab disconnects all instances of {@link AppMediator} connected
+   * @summary disconnectTab disconnects all instances of {@link AppMediator} connected
```

Comment:
@summary isn't a tsdoc tag, I think you're confusing with jsdoc. You can enable the eslint tsdoc plug in to check the syntax for you. Have a look in smoldot-test-utils or the connect packages for how to do it

Reply:
Took the @summary from `packages/connect/src/SmoldotProvider/SmoldotProvider.ts` that was added in the tsDocs there and I assumed that it was correct. 
As agreed I will add the TsDoc configuration in `connect` and `extension` workspace for now and correct all issues that the configuration will show